src/index.js

```diff
@@ -16,7 +16,6 @@ import postcss from 'rollup-plugin-postcss';
 import gzipSize from 'gzip-size';
 import brotliSize from 'brotli-size';
 import prettyBytes from 'pretty-bytes';
-import shebangPlugin from 'rollup-plugin-preserve-shebang';
```

Comment:
Oh good - I seriously spent about 30 mins trying to PR that project with updates. And I could not get rollup to like anything I was doing in that `transformChunk` hook. 👀